mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -425,6 +426,7 @@ CLASS aid net/minecraft/entity/Entity
 	METHOD r isTeammate (Laid;)Z
 	METHOD s isPartOf (Laid;)Z
 	METHOD t isSpectator ()Z
+	METHOD t onAttacked (Laid;)Z
```

Comment:
Looks like `handlePlayerAttack(Entity)` to me as this is only used by players and can skip the rest of the player attack logic